index.js

```diff
@@ -72,6 +73,19 @@ var CHUNK_NAMES = {
 };
 
 
+function parseMaterialNameChunk(buf) {
+  return {
+    materialName: fromAsciiz(buf)
+  };
+}
+
+
+function fromAsciiz(buf) {
```

Comment:
Could this function be made to work in `parseObjectChunk` ? If the source string really is an ASCIIZ string then its length will safely be usable as the buffer offset in the aforementioned function.